storage-node/src/api-spec/openapi.yaml

```
@@ -207,6 +207,19 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/DataStatsResponse'
+  /status:
+    get:
+      operationId: stateApi.getStatus
+      description: Returns json object describing current node status.
```

```suggestion
      description: Returns json object describing current node status.
      tags:
        - state
 ```
This will make the method part of `StateApi` in autogenerated client, instead of creating a new `DefaultApi` class

---

Addressed in [e545d72](https://github.com/Joystream/joystream/pull/4560/commits/e545d72a8d17c3f1f3c016cd2ef157b3935cc9a8)